### PR TITLE
Implemented greater/less than or equal too as a single condition

### PR DESCRIPTION
**Pull request by SamGodwin2**
### What is the context of this PR?
Author needs to be able to support writing inclusive comparators as part of the routing based on a number type answer. This PR implements these new conditions.

### How to review
Tests pass and routing is now possible using inclusive comparator signs.

### CODE REVIEW

File: tests/app/questionnaire/test_rules.py [hunk 1]
```diff
@@ -131,6 +131,27 @@ def test_evaluate_rule_not_equals_with_number(self):
...
+    
+    def test_evaluate_rule_greater_than_or_equals_with_number(self):
+        when = {
+            'value': 4,
+            'condition': 'greater than or equal to'
+        }
+
+        self.assertTrue(evaluate_rule(when, 4))
+        self.assertTrue(evaluate_rule(when, 5))
+        self.assertFalse(evaluate_rule(when, 3))
```
Comment (bitdivision) [inline tests/app/questionnaire/test_rules.py:144]: I know it's not done on the other tests, but we should probably be testing 'None' values.

File: app/questionnaire/rules.py [hunk 2]
```diff
@@ -60,29 +60,26 @@ def evaluate_condition(condition, answer_value, match_value):
...
-        result = answer_value is not None and answer_value != []
-    elif condition == 'not set':
-        result = answer_value is None or answer_value == []
-    elif condition == 'greater than' and answer_and_match and answer_value > match_value:
-        result = True
-    elif condition == 'less than' and answer_and_match and answer_value < match_value:
-        result = True
-
-    return result
+    conditions_dict = {
```
Comment (bitdivision) [inline app/questionnaire/rules.py]: Maybe call this `comparison_operators`

**Comment by bitdivision** [comment]
@SamGodwin2 Travis should pass now that the validator change has gone in. Happy to approve once the two comments are addressed.

**Review by bitdivision — APPROVED** [review]
:+1:

File: tests/functional/spec/features/routing/number.spec.js [hunk 3]
```diff
@@ -147,6 +147,40 @@ describe('Feature: Routing on a Number', function () {
           .getUrl().should.eventually.contain(IncorrectAnswerPage.pageName);
       });
     });
+
+    describe('Given I start number routing greater then or equal single condition survey', function () {
```
Comment (MebinAbraham) [inline tests/functional/spec/features/routing/number.spec.js]: `greater than or equal to`

File: tests/functional/spec/features/routing/number.spec.js [hunk 4]
```diff
@@ -147,6 +147,40 @@ describe('Feature: Routing on a Number', function () {
...
+
+      var NumberQuestionPage = require('../../../pages/features/routing/number/greater_than_or_equal/number-question.page');
+      var CorrectAnswerPage = require('../../../pages/features/routing/number/greater_than_or_equal/correct-answer.page');
+      var IncorrectAnswerPage = require('../../../pages/features/routing/number/greater_than_or_equal/incorrect-answer.page');
+
+      before(function () {
+        return helpers.openQuestionnaire('test_routing_number_greater_than_or_equal_single_condition.json');
+      });
+
+      it('When I enter a greater number, Then I should be routed to the correct page', function () {
```
Comment (MebinAbraham) [inline tests/functional/spec/features/routing/number.spec.js]: These texts don't really make sense...
`When I enter a great number than ...?`

Same of all others.

**Review by bitdivision — APPROVED** [review]

**Review by MebinAbraham — APPROVED** [review]

**Pull request merged**